src/UI/Components/Diagram/helpers.ts

```diff
@@ -440,3 +440,39 @@ export const findCorrespondingId = (
     attributeName,
   })).find(({ id }) => id === instanceAsTable.id);
 };
+
+/**
+ * Toggle the highlighting of a loose element in a diagram cell view.
+ * @param {dia.CellView} cellView - The cell view containing the element.
+ * @param {"add" | "remove"} kind - The action to perform, either "add" to add highlighting or "remove" to remove highlighting.
+ * @returns {void}
+ */
+export const toggleLooseElement = (
+  cellView: dia.CellView,
+  kind: "add" | "remove",
+): void => {
+  switch (kind) {
+    case "add":
+      highlighters.mask.add(cellView, "root", "loose_element", {
+        padding: 0,
+        className: "loose_element-highlight",
+        attrs: {
+          "stroke-opacity": 0.5,
+          "stroke-width": 5,
+          filter: "drop-shadow(3px 5px 2px rgb(0 0 0 / 0.4))",
+        },
+      });
+      break;
+    case "remove":
+      dia.HighlighterView.get(cellView, "loose_element")?.remove();
```

Comment:
same remark as higher up, this might end up in failing silently and not calling the method. Is this a behavior that you wanted?

Reply:
Similarly to comment above, theoretically it's possible to get undefined/null here, if there won't be highlighterView that would mean there is no red border(in this case, green in other place). Which here is undesired but that would meant that it was renamed in the line above.

Reply:
I don't know why i chose switch over if in that case though, I can switch(no pun intended) it in that case(no pun intended as well) :)